client/src/pages/Settings/Settings.tsx

```diff
@@ -81,30 +84,8 @@ export default function Settings(): JSX.Element {
             </Box>
           ))}
         </Grid>
-        <Grid xs={9} item>
-          <Switch>
-            <Route exact path="/profile/settings">
-              <Redirect to="/profile/settings/edit-profile" />
-            </Route>
-            <SettingsWrapper>
-              {settingsMenu.map((item) => (
-                <Route
-                  key={item.name}
-                  path={item.to}
-                  render={(props) =>
-                    cloneElement(item.component, {
-                      ...props,
-                      currentUser: loggedInUser,
-                      currentProfile: profile,
-                    })
-                  }
-                />
-              ))}
-            </SettingsWrapper>
-            <Route path="*">
-              <Redirect to="/not-found" />
-            </Route>
-          </Switch>
+        <Grid xs={12} md={9} item>
+          <DesktopAvailability />
```

Comment:
So, not sure why you deleted all this code and moved in into the DesktopAvailability, can we revert this. This is also not a proper name for this component since it houses ALL settings in the app.